api/build.gradle

```diff
@@ -6,7 +6,6 @@ def swaggerSourceFile = 'src/main/resources/workbench.yaml'
 def swaggerTargetFolder = 'src/generated/java'
 
 task generateApi {
-  delete("${projectDir}/$swaggerTargetFolder")
```

Comment:
This caused the generateApi task to run on every single build even if the files were up-to-date.

Reply:
The reason this was introduced is that stale code would get left over in the generated dir and cause problems if you changed a Swagger definition file.

I don't like having to generate the API every time, either... but then we better make it clear what procedure to follow if we're not going to do this. (The ideal would be: delete this only when one of our Swagger files changes.)

Reply:
I can live with having to do a clean build in this case. How about you?

Reply:
That's fine. You may want to update the README. The particular issue I believe is if you're removing a service or object from the Swagger file, it will still hang around until you do a clean.

Reply:
Added a `clean` command and noted this use case.